### PR TITLE
[WIP] refactor: made a single colors variable component

### PR DESCRIPTION
**Pull request by kush-x7**
### Made CSS variables of colors in file variables.css and used those variables in all CSS files.

Resolved issue number #308

### CODE REVIEW

File: app/styles/variables.css [hunk 1]
```diff
@@ -1,4 +1,66 @@
...
+  --slider-background: #CFD3FF;
+  --thumb-background: #A1D0FF;
+  --opaque-black: #000000b3;
+  --white: #ffffff;
+  --black: #000000;
+  --green: #008000;
+
   --dark-gray: #666;
   --off-white: #f0f2f5;
   --black: #000000f2;
```
Comment (rohan09-raj) [inline app/styles/variables.css:66]: Can we please remove these as they are already defined on the top ?

File: app/styles/variables.css [hunk 2]
```diff
@@ -1,4 +1,66 @@
...
+
+  /* Below this were the variables which was defined earlier */
+
+  --slider-colour: #424CC1;
+  --slider-background: #CFD3FF;
+  --thumb-background: #A1D0FF;
+  --opaque-black: #000000b3;
+  --white: #ffffff;
+  --black: #000000;
+  --green: #008000;
```
Comment (rohan09-raj) [inline app/styles/variables.css:62]: Are we using it anywhere ??

File: app/styles/navbar.css [hunk 3]
```diff
@@ -83,7 +83,7 @@ nav li a:hover {
   font-size: 0.87rem;
   flex-direction: column;
   justify-content: center;
-  background-color: #fffbfb;
+  background-color: var(--color-off-white);
   box-shadow: rgba(60, 64, 67, 0.3) 0px 1px 2px 0px, rgba(60, 64, 67, 0.15) 0px 2px 6px 2px;
```
Comment (rohan09-raj) [inline app/styles/navbar.css:87]: Can we please include these box shadow colors in the variables file too?

File: app/styles/variables.css (hunk 2 above)
Comment (rohan09-raj) [reply app/styles/variables.css:62]: If yes, please update it according to the convention you have used all over the place

File: app/styles/navbar.css (hunk 3 above)
Comment (kush-x7) [reply app/styles/navbar.css:87]: Sure Rohan. 
Now I will Write CSS Variables name according to our second approach and update them.

**Comment by kush-x7** [comment]
Made a new PR #321 . Now closing this one

**Pull request closed without merging**